homeassistant/components/daikin/sensor.py

```diff
@@ -96,10 +96,6 @@ def unit_of_measurement(self):
         """Return the unit of measurement."""
         return self._unit_of_measurement
 
-    def update(self):
-        """Retrieve latest state."""
-        self._api.update()
```

Comment:
How will this be replaced?

Reply:
The https://github.com/home-assistant/home-assistant/pull/21638/files#diff-3b8925f78f9aa3c942a136e17cf30eb9R118 updates all states for the device.

Reply:
We shouldn't rely on another platform polling the api in this platform. Ie even though the climate platform is calling `self._api.async_update` we can't rely on that here in the sensor platform. I think that's what you mean right?

Reply:
Yes that is what I mean. I'll revert the removal here then.